src/main/java/org/ays/emergency_application/model/request/EmergencyEvacuationApplicationRequest.java

```diff
@@ -76,22 +74,22 @@ public class EmergencyEvacuationApplicationRequest {
     @Valid
     private AysPhoneNumberRequest applicantPhoneNumber;
 
-
     @JsonIgnore
     @AssertTrue(message = "all applicant fields must be filled")
     @SuppressWarnings("This method is unused by the application directly but Spring is using it in the background.")
     private boolean isAllApplicantFieldsFilled() {
 
-        if (StringUtils.isEmpty(this.applicantFirstName) && StringUtils.isEmpty(this.applicantLastName) && this.applicantPhoneNumber == null) {
+        if (StringUtils.isEmpty(this.applicantFirstName) &&
+                StringUtils.isEmpty(this.applicantLastName) && this.applicantPhoneNumber == null) {
```

Comment:
Burayı da eski hâline alabilir miyiz?
```suggestion
        if (StringUtils.isEmpty(this.applicantFirstName) && StringUtils.isEmpty(this.applicantLastName) && this.applicantPhoneNumber == null) {
```

Reply:
![Screenshot 2025-01-18 192921](https://github.com/user-attachments/assets/85b732e4-d764-4e21-8d7f-6dd9a9fee852)

Hocam 3 defa bunları sizin attığınız şekilde commit ettim. Neden her defasında ben de yanlış gösteriyor yaa :(

Reply:
@m1erla https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html bu dokümanı takip edebilirsin